docs/README_de.md

```diff
@@ -0,0 +1,340 @@
+templural`${n} ist {other;many}`
+
+// die Angabe von zwei Werten ist für die Kategorien one, other and many
+templural`${n} ist {one;other;many}`
+```
+
+Die Standardpriorität (unabhängig vom Gebietsschema) ist:
+
+1. `other`
+2. `one`
+3. `two`
+4. `few`
+5. `many`
+6. `zero`
+
+Diese Standardpriorität wird so gefiltert, dass sie nur Kategorien des ausgewählten Gebietsschemas enthält, z. B. im Deutschen:
+
+1. `other`
+2. `one`
+
+Einige Sprachen haben möglicherweise eine andere Standardpriorität, siehe [locales.ts](https://github.com/nlepage/templural/blob/main/src/locales.ts).
+
+### Kategoriereihenfolge
+
+Die Kategoriereihenfolge legt die Reihenfolge fest, in der die Kategorien bewertet werden (unabhängig von ihrer Priorität).
+
+Wenn die Reihenfolge:
+
+1. `many`
+2. `other`
+3. `one`
+
+Dann:
+
+```js
+// mit zwei Werten ist other dann one (many hat eine niedrigere Priorität)
+templural`${n} ist {other;one}`
+
+// mit drei Werten ist many dann other danach one
+templural`${n} ist {many;other;one}`
+```
+
+Die Standardpriorität (unabhängig vom Gebietsschema) ist:
+
+1. `zero`
+2. `one`
+3. `two`
+4. `few`
+5. `many`
+6. `other`
+
+Diese Standardreihenfolge wird so gefiltert, dass sie nur Kategorien des jeweiligen Gebietsschemas enthält, z. B. im Deutschen:
+
+1. `one`
+2. `other`
+
+Einige Sprachen haben möglicherweise eine andere Standardreihenfolge, siehe [locales.ts](https://github.com/nlepage/templural/blob/main/src/locales.ts).
+
+### Kategorie Fallback
+
+Bestimmte Sprachen benötigen möglicherweise eine Kategorie, um auf eine andere Kategorie zurückgreifen zu können.
+
+Zum Beispiel im Französischen, `many` verhält sich wie eine Unterkategorie von `other`, was bedeutet `many` muss zurückfallen auf `other`:
+
+```js
+templural`${1000000} ist {one;other;many} und fällt zurück auf {one;other}`
+// ➔ "1000000 ist many and fällt zurück auf other"
+```
+
+Einige Sprachen können Standard-Fallbacks haben, siehe [locales.ts](https://github.com/nlepage/templural/blob/main/src/locales.ts).
+
+## ❓ FAQ
+
+### Was ist mit negativen Zahlen, Gleitkommazahlen oder anderen Werten?
+
+templural kümmert sich nicht darum, es verlässt sich auf [Intl.PluralRegeln](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/PluralRules).
+
+### Sind neue Funktionen geplant?
+
+Im Moment nicht.
+
+templural ist einfach und dumm, und so wird es wahrscheinlich auch bleiben.
+
+### Haben Sie noch Fragen?
+
+Benutzen Sie [Discussions tab](https://github.com/nlepage/templural/discussions).
+
+## Autoren und Mitwirkende
+
+Ein besonderer Dank geht an [Valentin Cocaud](https://github.com/EmrysMyrddin) und [Yoann Prot](https://github.com/Taranys) für ihre ursprüngliche Idee von templural.
+
+👤 **Nicolas Lepage**
+
+* Twitter: [@njblepage](https://twitter.com/njblepage)
+* GitHub: [@nlepage](https://github.com/nlepage)
+
+👩 **SvA1**
+
+* GitHub: [@SvA1](https://github.com/SvA1)
```

Comment:
Sorry I've refactored this section.

There are a few things that cannot be translated:
 - the section's title must be "Contributors ✨"
 - the HTML between the comments

However you can translate the two sentences:
 - Thanks goes to these wonderful people
 - This project follows...

```suggestion
## Contributors ✨

<table>
  <tr>
    <td align="center"><a href="https://github.com/SvA1"><img src="https://avatars.githubusercontent.com/u/19571875?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Nicolas Lepage</b></sub></a><br /><a href="https://github.com/nlepage/templural/commits?author=nlepage" title="Author">✍️</a></td>
  </tr>
</table>

Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/docs/en/emoji-key)):

<!-- ALL-CONTRIBUTORS-LIST:START - Do not remove or modify this section -->
<!-- prettier-ignore-start -->
<!-- markdownlint-disable -->
<table>
  <tr>
    <td align="center"><a href="https://github.com/EmrysMyrddin"><img src="https://avatars.githubusercontent.com/u/3855602?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Valentin Cocaud</b></sub></a><br /><a href="#ideas-EmrysMyrddin" title="Ideas, Planning, & Feedback">🤔</a></td>
    <td align="center"><a href="https://github.com/Taranys"><img src="https://avatars.githubusercontent.com/u/4621525?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Yoann Prot</b></sub></a><br /><a href="#ideas-Taranys" title="Ideas, Planning, & Feedback">🤔</a></td>
    <td align="center"><a href="https://github.com/SvA1"><img src="https://avatars.githubusercontent.com/u/41849274?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Svetlana A.</b></sub></a><br /><a href="https://github.com/nlepage/templural/commits?author=SvA1" title="Documentation">📖</a></td>
  </tr>
</table>

<!-- markdownlint-restore -->
<!-- prettier-ignore-end -->

<!-- ALL-CONTRIBUTORS-LIST:END -->

This project follows the [all-contributors](https://github.com/all-contributors/all-contributors) specification. Contributions of any kind welcome!
```